src/helpers/library.ts

```diff
@@ -146,11 +146,33 @@ async function uninstall({ appName, handleGameStatus, t }: UninstallArgs) {
     type: 'warning'
   }
 
-  const { response } = await ipcRenderer.invoke('openMessageBox', args)
+  let linuxArgs
+  if ((await getPlatform()) === 'linux') {
+    const wineprefix = (await getGameSettings(appName)).winePrefix
+
+    linuxArgs = {
+      checkboxLabel: [
+        t(
+          'gamepage:box.uninstall.checkbox',
+          "Would you like to remove the prefix aswell? This can't be undone."
+        ),
+        `${t(
+          'gamepage:box.uninstall.checkbox_prefix',
+          'Prefix'
+        )}: ${wineprefix}`
+      ].join('\n'),
+      checkboxChecked: true
```

Comment:
I think should be false by default.
```suggestion
      checkboxChecked: false
```